frontend/test/metabase/scenarios/binning/qb-explicit-joins.cy.spec.js

```diff
@@ -0,0 +1,328 @@
+import { restore, popover } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const {
+  ORDERS_ID,
+  ORDERS,
+  PEOPLE_ID,
+  PEOPLE,
+  PRODUCTS_ID,
+  PRODUCTS,
+} = SAMPLE_DATASET;
+
+describe("scenarios > binning > from a saved QB question with explicit joins", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    cy.createQuestion({
+      name: "QB Binning",
+      query: {
+        "source-table": ORDERS_ID,
+        joins: [
+          {
+            fields: [
+              ["field", PEOPLE.LONGITUDE, { "join-alias": "People" }],
+              [
+                "field",
+                PEOPLE.BIRTH_DATE,
+                { "temporal-unit": "default", "join-alias": "People" },
+              ],
+            ],
```

Comment:
Choosing only the specific fields that we need for this test for each of the tables.